nimterop.nimble

```diff
@@ -59,20 +59,24 @@ when (NimMajor, NimMinor, NimPatch) >= (0, 19, 9):
     fmt"{currentSourcePath}".parentDir.parentDir.parentDir
 
 proc runNimDoc() =
-  execCmd &"nim doc -o:{htmldocsDir} --project --index:on nimterop/all.nim"
-  execCmd &"nim buildIndex -o:{htmldocsDir}/theindex.html {htmldocsDir}"
-  when declared(getNimRootDir):
-    #[
-    this enables doc search, works at least locally with:
-    cd {htmldocsDir} && python -m SimpleHTTPServer 9009
-    ]#
-    execCmd &"nim js -o:{htmldocsDir}/dochack.js {getNimRootDir()}/tools/dochack/dochack.nim"
+  when not defined(windows): # pending https://github.com/nim-lang/Nim/pull/10629
+    execCmd &"nim doc -o:{htmldocsDir} --project --index:on nimterop/modules.nim"
+    execCmd &"nim buildIndex -o:{htmldocsDir}/theindex.html {htmldocsDir}"
+    when declared(getNimRootDir):
+      #[
+      this enables doc search, works at least locally with:
+      cd {htmldocsDir} && python -m SimpleHTTPServer 9009
```

Comment:
Works online too, can remove comment.

Reply:
it doesn't work locally without `python -m SimpleHTTPServer`:
```
open build/htmldocs/theindex.html
```

![image](https://user-images.githubusercontent.com/2194784/52685381-5c806500-2efe-11e9-8291-c063a55cf644.png)

Reply:
well, removed it anyways